### ISSUE TITLE
Clear and lock down PWR parks

### ISSUE THREAD

**Issue by nateirwin** [nationalparkservice/places-data#565]
I think we should discuss this before moving forward. I'd like to do a basic audit of what data is available in the PWR web services to ensure we'll have decent maps for parks going forward. I'm specifically worried about points of interest. It seems like the regional office, to this point, hasn't focused much time on this dataset, but it is one of the more important datasets for Park Tiles.

**Comment by ryanjhodge** [comment]
Does this mean PWR datasets will not be editable via the Places Editor?

**Comment by nateirwin** [comment]
Yeah. Sandy in the PWR office wants all the data to funnel up through her GIS datasets/services. We can, however, keep a park unlocked if the park itself prefers to use Places Editor.
